301-parameterized-linked-templates/detail/complexvariables.md

```diff
@@ -0,0 +1,91 @@
+# Using Variables to Centralize Configurable Elements
+
+The variables section in this sample's [azuredeploy.json](../azuredeploy.json#L192-L341) is used mostly to create lookup tables that are used to alter the deployment configuration based on the user's selections.  Several examples are described below.
+
+The variable `AvailabilitySetFaultDomain` is used to allow selection of the number of fault domains for Availability Sets based on the maximum number for the deployment region:
```

Comment:
this pattern is a little verbose... you can simply add the location that contain 3 FD and omit all others.  Then if the var contains the location you have 3, else 2... will make it easier to incorporate AzureGov too

Reply:
Fixed, removed all the regions that only have 2 fault domains and using the logic for the next variable to default to 2.